WebContent/css/styles.css

```diff
@@ -37,11 +37,6 @@ body {
     width: 100%;
 }
 
-/* Override inline width style */
-.dataset-qualifier-field{
```

Comment:
one reason for style not working, 
1)  wrong spelling - dataset-qualifier-field vs correct datasets-qualifier-field
2) wrong selector, class selector vs css selector, should be #dataset-qualifier-field 
3) unable to override width or flex-grow property from styles.css for some reason.
     inline styles are working